app/src/main/java/edu/uc/hickmadc/customnotifications/adapters/GroupsAdapter.kt

```diff
@@ -12,9 +12,7 @@ import edu.uc.hickmadc.customnotifications.dto.Group
 class GroupsAdapter(private val dataSet: ArrayList<Group>) :
     ListAdapter<Group, RecyclerView.ViewHolder>(GroupDiffCallback()) {
 
-    /**
-     * Creates list rows / views
-     */
+    // Creates list rows / views
```

Comment:
This is going in the wrong direction.  Previous syntax is closer to proper KDoc.  This is inline, which doesn't belong above a method.